gnd/src/main/java/com/google/android/gnd/ui/offlinebasemap/OfflineBaseMapsFragment.java

```diff
@@ -66,6 +66,17 @@ public View onCreateView(
     recyclerView.setAdapter(offlineBaseMapListAdapter);
 
     viewModel.getOfflineAreas().observe(getViewLifecycleOwner(), offlineBaseMapListAdapter::update);
+    viewModel
```

Comment:
Rather than subscribing and imperatively updating the visibility, can we instead expose an LiveData<Integer> that is derived from the getOfflineAreas() stream? We've started doing this elsewhere and I think we got the best of both worlds - readable code, and purely functional impl. Wdyt?

Reply:
0d480ff